docs/index.md

```diff
@@ -51,7 +51,7 @@ The application is a standalone binary capable of performing RTL-to-GDSII SoC de
 from logic synthesis and floorplanning through detailed routing with metal fill insertion,
 signoff parasitic extraction and timing analysis.
 
-See documentation for the application [here](../README.md).
+See documentation for the application [here](main/README.md).
```

Comment:
Page not found

Reply:
this will be resolved correctly in readthedocs.